src/models/media.ts

```diff
@@ -21,8 +21,38 @@ export interface Media {
   license: string
   license_version: string
   license_url: string
+  attribution: string // TODO: check if it exists for all media items
 
   frontendMediaType: SupportedMediaType
 
+  provider: string
+  source?: string
+
+  detail_url: string
+  related_url: string
+
   tags: Tag[]
 }
+
+export interface ImageDetail extends Media {
+  frontendMediaType: 'image'
+
+  fields_matched?: string[]
+}
+
+export interface AudioDetail extends Media {
+  frontendMediaType: 'audio'
+
+  thumbnail?: string
+  audio_set?: string
+  genres?: string[]
+  duration?: number
+  bit_rate?: number
+  sample_rate?: number
+  alt_files?: { provider: string; filetype: string }[]
+  filetype?: string
+  peaks?: number[]
```

Comment:
I think some of these are inaccurate, either their not optional properties (but actually required and nullable instead) or they're always present but sometimes just empty (like `peaks` is a required property but is sometimes an empty array).

It's not a big deal though, in particular with the nullability, it'll just make writing code using these types more annoying becuase we'll have to add null checks in places that we don't need them (like on the array types).

`thumbnail` can be moved up to `Media` too as it's on both Audio and Images.